templates/common/infra/terraform/core/gateway/apim/apim-api-policy.xml

```diff
@@ -0,0 +1,92 @@
+<!-- Policy configuration for the API. Explore other sample policies at https://learn.microsoft.com/en-us/azure/api-management/policies/ -->
```

Comment:
Is this file the same as the one used for bicep?  If so, then we could put it in a common location that can be used for both terraform and bicep.

Reply:
We currently put it in `/template/common/infra/gateway/apim/apim-api-policy.xml`. Do you think it's okay?